File: tests/fixtures/e2e/dotnet/good.cs
using Temporalio.Workflows;

[Workflow]
public class GoodWorkflow
{
    [WorkflowRun]
    public async Task<string> RunAsync(MyInput input)
    {
        var result = await Workflow.ExecuteActivityAsync(
            () => MyActivities.DoWork(input),
            new() { StartToCloseTimeout = TimeSpan.FromMinutes(5) });
        return result;
    }
}

File: tests/fixtures/e2e/typescript/good.ts
import { proxyActivities } from '@temporalio/workflow';
import type { MyActivities } from './activities';

const { doWork } = proxyActivities<MyActivities>({
  startToCloseTimeout: '5m',
});

export async function goodWorkflow(input: WorkflowInput): Promise<string> {
  return await doWork(input);
}
